dbms/src/Flash/Executor/toRU.h

```diff
@@ -18,7 +18,7 @@
 
 namespace DB
 {
-using RU = UInt64;
+using RU = double;
```

Comment:
https://github.com/pingcap/tiflash/blob/569b5672dfcdee6da25e6e73d89675010b4339f8/dbms/src/Flash/Coprocessor/DAGContext.cpp#L440
If RU is changed to double, can the ceil here be removed? Because there is no need to round up